tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -269,6 +269,43 @@
             - openvswitch3.1
     EOF
 
+- name: prepare adopted EDPM workloads to use Ceph backend for Cinder, if configured so
+  no_log: "{{ use_no_log }}"
+  when: cinder_volume_backend == "ceph" or cinder_backup_backend == "ceph"
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc patch osdpns/openstack --type=merge --patch "
+    spec:
+      services:
+        - repo-setup
+        - download-cache
+        - bootstrap
+        - configure-network
+        - validate-network
+        - install-os
+        - configure-os
+        - run-os
+        - reboot-os
+        - install-certs
+        - ceph-client
+        - libvirt
+        - nova-compute-extraconfig
+        - ovn
+        - neutron-metadata
+      nodeTemplate:
+        extraMounts:
+        - extraVolType: Ceph
+          volumes:
+          - name: ceph
+            secret:
+              secretName: ceph-conf-files
+          mounts:
+          - name: ceph
+            mountPath: "/etc/ceph"
```

Comment:
I wonder if this would fail for aeee when it will be running tasks for services listed before the ceph-client service

Reply:
this seems like worked OK